README.md

```diff
@@ -1,2 +1,30 @@
 # LinQuadOptInterface.jl
-Intermediate wrapper for MOI of GLPK, CPLEX, Gurobi and Xpress
+
+**Attention**: This is an intermediary layer used by some implementations of the MathOptInterface.
+
+LinQuadOptInterface (LQOI) is designed to make it easier for low-level wrapper designed to bridge low-level Integer Linear and Quadratic solvers to implement the [MathOptInterface](https://github.com/JuliaOpt/MathOptInterface.jl) (MOI) interface. It provides access to many MOI functionalities mainly related to problem modifications:
+
+1. add constraints/variables 1-by-1 and in batches
+2. remove constraints/variables
+3. change constraint coefficients
+4. change constraint type
+5. change constraint rhs
+6. change variable bounds and type
```

Comment:
Maybe add a sentence making it explicit that using LQOI is entirely optional and comes with a trade-off of introducing an extra abstraction layer between the solver and MOI.